app/views/catalog/_facet_pagination.html.erb

```diff
@@ -1,3 +1,13 @@
+    <% if @facet.index_pagination && params["facet.sort"] == 'index' %>
+      <div class="alpha-filter">
+        <%= t('blacklight.search.facets.filter') %>:
+        <%= link_to(t('blacklight.search.facets.clear'), @pagination.params_for_resort_url('index', params.except(:"facet.prefix")), :data => {:ajax_modal => "preserve"}) %>
+        <% ('A' .. 'Z').to_a.each do |letter| %>
```

Comment:
yes, facet.prefix is case sensitive, both the parameter itself and its value.  it is a solr feature to allow facet filtering, and  i could not find an easy way to have it be case insensitive